src/random.c

```diff
@@ -35,7 +35,7 @@ static int uv__random(void* buf, size_t buflen) {
   rc = uv__random_readpath("/dev/urandom", buf, buflen);
 #elif defined(_AIX)
   rc = uv__random_readpath("/dev/random", buf, buflen);
-#elif defined(__APPLE__) || defined(__OpenBSD__)
+#elif defined(__APPLE__) || defined(__OpenBSD__) || (defined(__ANDROID_API__) && __ANDROID_API__ >= 28)
```

Comment:
Long line, should be <= 80 columns.

You don't need to check the API version because libuv will fall back to /dev/urandom when `getentropy` isn't dlsym-able but I don't have strong feelings one way or the other.